### ISSUE TITLE
react-tree-orgchart : Spinning round with message "Loading Organization Chart ..."

### ISSUE THREAD

**Issue by tienou** [pnp/sp-dev-fx-webparts#1339]
## Sample
react-tree-orgchart

## Authors
@joaojmendes 

## Expected or Desired Behavior
**When Show only my team setting is set to false :**
![image](https://user-images.githubusercontent.com/57989387/84393687-2c34fd80-abfc-11ea-8dc7-10f286a8ed49.png)
Spinning round with message "Loading Organization Chart ..." and nothing Loading

## Steps to Reproduce
1. Add Tree-OrgChart Webpart
1. my team setting is set to false

# Environment Details
Target environment : SharepointOnline

@mpaukkon :
I noticed same behavior in our production tenant, when Show only my team setting is set to false. 
I investigated a little bit and found out that web part actually fetches all profiles in your organization's structure. It will generate a lot of calls to getpropertiesfor REST endpoint.

_Originally posted by @mpaukkon in https://github.com/pnp/sp-dev-fx-webparts/issues/1324#issuecomment-641705298_

**Comment by tienou** [comment]
`sp-pages-assembly_fr-fr_02f90020ea7c6f37b6dbab8a27342512.js:1 [Deprecation] chrome.loadTimes() is deprecated, instead use standardized API: Paint Timing. https://www.chromestatus.com/features/5637885046816768.
e._checkTTI @ sp-pages-assembly_fr-fr_02f90020ea7c6f37b6dbab8a27342512.js:1
publiccdn.sharepointonline.com/genesienne.sharepoint.com/sites/AppCatalog/ClientSideAssets/4d7c1cea-0ef5-4143-88f6-43411d978b2f/portal-footer-application-customizer_6bd076db7f04985da845b9bb1c84c17c.js:29 SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at t.<anonymous> (publiccdn.sharepointonline.com/genesienne.sharepoint.com/sites/AppCatalog/ClientSideAssets/4d7c1cea-0ef5-4143-88f6-43411d978b2f/portal-footer-application-customizer_6bd076db7f04985da845b9bb1c84c17c.js:29)
    at i (publiccdn.sharepointonline.com/genesienne.sharepoint.com/sites/AppCatalog/ClientSideAssets/4d7c1cea-0ef5-4143-88f6-43411d978b2f/portal-footer-application-customizer_6bd076db7f04985da845b9bb1c84c17c.js:29)
    at Object.next (publiccdn.sharepointonline.com/genesienne.sharepoint.com/sites/AppCatalog/ClientSideAssets/4d7c1cea-0ef5-4143-88f6-43411d978b2f/portal-footer-application-customizer_6bd076db7f04985da845b9bb1c84c17c.js:29)
    at a (publiccdn.sharepointonline.com/genesienne.sharepoint.com/sites/AppCatalog/ClientSideAssets/4d7c1cea-0ef5-4143-88f6-43411d978b2f/portal-footer-application-customizer_6bd076db7f04985da845b9bb1c84c17c.js:29)
publiccdn.sharepointonline.com/genesienne.sharepoint.com/sites/AppCatalog/ClientSideAssets/4d7c1cea-0ef5-4143-88f6-43411d978b2f/portal-footer-application-customizer_6bd076db7f04985da845b9bb1c84c17c.js:29 Current site is not part of an hub and the footer will fallback to local list of links.
/sites/Intranet/_api/web/lists/getByTitle('PnP-PortalFooter-Links')/items?$select=Title,PnPPortalLinkGroup,PnPPortalLinkUrl&$top=100&$orderby=PnPPortalLinkGroup%20asc,Title%20asc:1 Failed to load resource: the server responded with a status of 404 ()
sp-pages-assembly_fr-fr_02f90020ea7c6f37b6dbab8a27342512.js:1 Uncaught (in promise) Error: Failed to create application customizer 'ClientSideExtension.ApplicationCustomizer.df889434-1b1c-4f5b-ada3-086d948e4270'. Error information is 'Extension failed to initialize for componentId "df889434-1b1c-4f5b-ada3-086d948e4270".'.
    at sp-pages-assembly_fr-fr_02f90020ea7c6f37b6dbab8a27342512.js:1
    at async Promise.all (index 0)
spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-pages-search~sp-pages-search-b~f46164af_none_999fe2a950fe0c832c5f.js:1 Uncaught (in promise) undefined
spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-pages-search_none_696f1bf73d675a1e3a2a.js:1 Some icons were re-registered. Applications should only call registerIcons for any given icon once. Redefining what an icon is may have unintended consequences. Duplicates include: 
GlobalNavButton, ChevronDown, ChevronUp, Edit, Add, Cancel, More, Settings, Mail, Filter (+ 1791 more)
r @ spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-pages-search_none_696f1bf73d675a1e3a2a.js:1
Organnigramme(1).aspx:1 A cookie associated with a cross-site resource at https://webshell.suite.office.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
Organnigramme(1).aspx:1 Access to XMLHttpRequest at 'https://waconatm.officeapps.live.com/apc/trans.gif?10fa38fd418198e49de5b505c4f43a07' from origin 'https://genesienne.sharepoint.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
waconatm.officeapps.live.com/apc/trans.gif?10fa38fd418198e49de5b505c4f43a07:1 Failed to load resource: net::ERR_FAILED
tree-org-chart-web-part_977b7aef085049e621f95ae42be9d22e.js:1 Uncaught (in promise) TypeError: Cannot read property 'web' of undefined
    at e.get [as baseUrl] (tree-org-chart-web-part_977b7aef085049e621f95ae42be9d22e.js:1)
    at tree-org-chart-web-part_977b7aef085049e621f95ae42be9d22e.js:1
    at new Promise (<anonymous>)
    at b (tree-org-chart-web-part_977b7aef085049e621f95ae42be9d22e.js:1)
    at t.toRequestContext (tree-org-chart-web-part_977b7aef085049e621f95ae42be9d22e.js:1)
    at tree-org-chart-web-part_977b7aef085049e621f95ae42be9d22e.js:1
6/sites/Intranet/CDN/directory-web-part_1e9666b136ffd91d7d6d5082e8cb30b9.js:10 ƒ t(r){var n=e.call(this,r)||this;return n._isUnMounted=!1,n._bootstrapLivePersonaCard=function(e,r){return!t._isLivePersonaCardFlightEnabled()||ht.isMobileWebView()?Promise.resolve():t.LivePersonaCard…
4spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-suite-nav-search-common_none_d159586d9106e00bed4a.js:1 Uncaught (in promise) Error: Can not set initial state more than once.
    at f (spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-suite-nav-search-common_none_d159586d9106e00bed4a.js:1)
    at Module.u (spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-suite-nav-search-common_none_d159586d9106e00bed4a.js:1)
    at Module.C (spoprod-a.akamaihd.net/files/sp-client/chunk.sp-suite-nav-search-common_none_290bee60d1a6bc6b0abf.js:1)
    at sp-pages-assembly_fr-fr_02f90020ea7c6f37b6dbab8a27342512.js:1
45/sites/Intranet/CDN/directory-web-part_1e9666b136ffd91d7d6d5082e8cb30b9.js:10 ƒ t(r){var n=e.call(this,r)||this;return n._isUnMounted=!1,n._bootstrapLivePersonaCard=function(e,r){return!t._isLivePersonaCardFlightEnabled()||ht.isMobileWebView()?Promise.resolve():t.LivePersonaCard…
lpcres.delve.office.com/lpc/versionless/livepersonacard_with-react_73c84b8ef8f6a35b3f2a.js:53 Some icons were re-registered. Applications should only call registerIcons for any given icon once. Redefining what an icon is may have unintended consequences. Duplicates include: 
GlobalNavButton, ChevronDown, ChevronUp, Edit, Add, Cancel, More, Settings, Mail, Filter (+ 4869 more)
r @ lpcres.delve.office.com/lpc/versionless/livepersonacard_with-react_73c84b8ef8f6a35b3f2a.js:53
/sites/Intranet/CDN/directory-web-part_1e9666b136ffd91d7d6d5082e8cb30b9.js:10 LivePersonaCard Open
4spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-suite-nav-search-common_none_d159586d9106e00bed4a.js:1 Uncaught (in promise) Error: Can not set initial state more than once.
    at f (spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-suite-nav-search-common_none_d159586d9106e00bed4a.js:1)
    at Module.u (spoprod-a.akamaihd.net/files/sp-client/chunk.vendors~sp-suite-nav-search-common_none_d159586d9106e00bed4a.js:1)
    at Module.C (spoprod-a.akamaihd.net/files/sp-client/chunk.sp-suite-nav-search-common_none_290bee60d1a6bc6b0abf.js:1)
    at sp-pages-assembly_fr-fr_02f90020ea7c6f37b6dbab8a27342512.js:1
2DevTools failed to load SourceMap: Could not load content for https://static.sharepointonline.com/bld/_layouts/15/16.0.20203.12005/ActivityMonitor.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/crossdomainproxyiframe.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/tokenfactoryiframe.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/bootstrapper.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/responsive.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/core.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/searchbox.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/plus.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/sb_strings.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/notifications.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://oneshellprcorp.blob.core.windows.net/oneshellpr/20200607.1/searchux.map: HTTP error: status code 403, net::ERR_HTTP_RESPONSE_CODE_FAILURE`

**Comment by tienou** [comment]
Hello @hugoabernier ,
Can you assign the issue to @joaojmendes ?
Thanks

**Comment by hugoabernier** [comment]
@tienou all work in this repository is done on a volunteer basis. I'll assign the issue when he volunteers for it.

**Comment by tienou** [comment]
> @tienou all work in this repository is done on a volunteer basis. I'll assign the issue when he volunteers for it.

Ok, I understand, thanks for the explanation

**Comment by PopWarner** [comment]
Hi @tienou,

We believe that PR #1338 has fixed this issue. Can you please confirm and if you still see the issue, please open a new issue. 

Thank you for your community collaboration. 💜🧡
